src/components/TextInput/TextInput.js

```diff
@@ -29,12 +29,12 @@ class TextInput extends React.Component {
     label: PropTypes.string.isRequired,
     width: PropTypes.number,
     value: PropTypes.any
-  }
+  };
 
   componentDidMount = () => {
-    valueTest(this.inputRef.current)
+    runValidation(this.inputRef.current)
```

Comment:
we are not doing any validation in react-spark

Reply:
removed